app/uk/gov/hmrc/trackingconsentfrontend/assets/package.json

```diff
@@ -10,7 +10,7 @@
     "build": "webpack --mode production",
     "lint": "stylelint \"**/*.scss\" && standardx **/*.ts",
     "fixlint": "stylelint --fix \"**/*.scss\" && standardx --fix **/*.ts",
-    "test": "npm run lint && jest --collectCoverage",
+    "test": "jest --collectCoverage && npm run lint",
```

Comment:
I've swapped these to allow tests to run regardless of whether linting has passed.  I hit this use case a number of times before changing this - particularly while commenting out code to see which tests break and being forced to remove unused imports before seeing the test results.